pkg/config/deployment.go

```diff
@@ -370,6 +372,21 @@ type SecretEncryption struct {
 	DecryptionTargets []string `json:"decryptionTargets"`
 }
 
+// Notification represents the way to send to users.
+type Notification struct {
+	// List of users to be notified for each event.
+	Mentions []NotificationMention `json:"mentions"`
+}
+
+type NotificationMention struct {
+	// The event to be notified to users
+	Event model.Stage `json:"event"`
```

Comment:
Possibly we can use `model.NotificationEventType`

Reply:
But actually `model.NotificationEventType` is int32. So you can simple use `string` here. Then let's add a `Validate()` function to check if it's within `model.NotificationEventType_value` or not.

Reply:
7e2498b